tests/conftest.py

```diff
@@ -319,6 +345,47 @@ def _create_task(task_name=""):
         yield _create_task
 
 
+@pytest.fixture(scope="function")
+def watch_dog_task(request, init_kwargs, any_x_series_device) -> nidaqmx.system.WatchdogTask:
+    """Gets a task instance."""
+    # set default values used for the initialization of the task.
+    init_args = {
+        "device_name": any_x_series_device.name,
```

Comment:
FYI, it's kind of weird for this fixture to always require an X Series device, but I suppose it's not a big deal because we don't explicitly test watchdog with cDAQ or static DIO.